src/agent/Core/ApplicationPool/Process.h

```diff
@@ -102,6 +101,12 @@ class Process {
 public:
 	static const unsigned int MAX_SOCKETS_ACCEPTING_HTTP_REQUESTS = 3;
 
+	/**
```

Comment:
If you need to access a field from one of the `Group::findXXX()` functions then there's no need to hoist that field to 'public'. You can `friend class Group`.